src/simularium/OctopusClient.ts

```diff
@@ -0,0 +1,76 @@
+import { TrajectoryType } from "../constants.js";
+import { WebsocketClient, NetMessageEnum } from "./WebsocketClient.js";
+
+interface IOctopusServices {
```

Comment:
Is there an expectation of more than one different kind of implementation of IOctopusServices?  If not, then the extra layer is not needed.